src/Asset/Service/AssetService.php

```diff
@@ -86,6 +86,7 @@ public function getAssets(ElementParameters $parameters): Collection
         );
 
         $assetQuery->orderByPath('asc');
+        $assetQuery->setUser($this->securityService->getCurrentUser());
```

Comment:
Not sure about this one, since I think we use the getAssets also for async calls, where we do not have a user no?

Reply:
I did not find any usage of this in the async call, only in the tree endpoint.

Reply:
> I did not find any usage of this in the async call, only in the tree endpoint. I think we query DB for async calls?

Hmm yeah not sure, I thought maybe in the handlers somewhere :) but if that is the case we can go ahead and merge it